permissionless-batches/conf/coordinator/coordinator_run.sh

```
@@ -0,0 +1,127 @@
+  echo "Checksum verification passed for $filename ✅"
+}
+
+# Download and verify asset files
+curl -o "$ASSETS_CHECKSUM_FILE" -L "$ASSETS_CHECKSUM_URL"
+for url in "${ASSETS_URLS[@]}"; do
+  download_and_verify "$url" "$ASSETS_DIR" "$ASSETS_CHECKSUM_FILE"
+done
+
+# Download OpenVM files (No checksum verification, but skips if file exists)
+for url in "${OPENVM_URLS[@]}"; do
+  dest_subdir="$OPENVM_DIR/$(basename $(dirname "$url"))"
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)